<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="8" column="0">
    <widget class="QGroupBox" name="groupBoxSize">
     <property name="title">
      <string>Window Size</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="radioReset">
        <property name="text">
         <string>Reset Size</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioMaximize">
        <property name="text">
         <string>Maximized</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioFull">
        <property name="text">
         <string>Full Screen</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="11" column="0" colspan="5">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="spinDefault">
     <property name="maximum">
      <number>1440</number>
     </property>
     <property name="singleStep">
      <number>5</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QWidget" name="fgColor" native="true">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="QCheckBox" name="cb_autostart">
     <property name="text">
      <string>Autostart countdown at launch</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="1">
    <widget class="QGroupBox" name="groupBoxHiding">
     <property name="title">
      <string>Hide items at launch</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="cb_no_menu">
        <property name="text">
         <string>Hide menu</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_no_toolbar">
        <property name="text">
         <string>Hide toolbar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QPushButton" name="pushReset">
     <property name="text">
      <string>Reset settings</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Change digit color:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Default start time</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>spinDefault</tabstop>
  <tabstop>pushReset</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
